openedx/adg/lms/applications/tests/conftest.py

```diff
@@ -94,9 +105,58 @@ def expired_course_overview(current_time):
     )
 
 
-@pytest.fixture()
-def test_user():
+@pytest.fixture(name='test_user')
+def testing_user():
     """
     Fixture which returns a user
     """
-    return UserFactory()
+    return UserFactory(username=USERNAME, password=PASSWORD)
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='user')
+def user_with_application_hub_and_extended_profile_fixture(test_user):
+    """
+    Create a test user and their corresponding ApplicationHub
```

Comment:
We are not creating a test user here, rather a user is passed in parameters to the fixture